.github/workflows/ci.yml

```
@@ -0,0 +1,17 @@
+name: CI
+
+on:
+  push:
+    branches: [ trunk ]
+  pull_request:
+    branches: [ trunk ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container:
+      image: wordpressmobile/android-build-image
+    steps:
+    - uses: actions/checkout@v2
+    - name: Test
+      run: ./gradlew :plugin:check
```

Would this have worked with another indentation level? That's usually how I see arrays in Markdown 🤔 It clearly works from a parsing point of view this way, so no need to worry–just me wondering 😅 

```suggestion
    steps:
      - uses: actions/checkout@v2
      - name: Test
        run: ./gradlew :plugin:check
```

---

I think it'd. I am working on another PR, I'll make the change in that if I don't forget it 🙈

---

🤦 I thought you already merged this. I'll just accept the suggestion :)